File: MAP/Lab 2/ToyInterpreter/src/view/commands/PrintCommand.java
package view.commands;


import controller.Controller;
import model.expressions.ArithExp;
import model.expressions.ConstExp;
import model.expressions.IExp;
import model.expressions.VarExp;
import model.statements.*;

import java.util.Scanner;

public class PrintCommand extends Command {

    public PrintCommand(String key, String description, Controller controller) throws Exception {
        super(key, description, controller);
    }

    @Override
    public void execute() throws Exception {
        printPrograms();
    }

    public void printPrograms() {
        System.out.println( ""  + controller.getPrograms().toString()  );
    }

}

File: MAP/Lab10/ConsoleApp1/ConsoleApp1/View/commands/PrintCommand.cs
using System;
using System.Collections.Generic;
using System.Text;
using ToyInterpreter.Controller;

namespace ToyInterpreter.View.commands
{
    class PrintCommand: Command
    {
        public PrintCommand(String key, String description, Controller.Controller controller): base(key, description, controller)
        {
            
        }

        public override void execute()
        {
            printPrograms();
        }

        public void printPrograms()
        {
            Console.WriteLine( controller.ToString() );
        }
    }
}
